examples/gno.land/r/demo/boards2/public.gno

```diff
@@ -57,21 +72,38 @@ func CreateReply(bid BoardID, threadID, replyID PostID, body string) PostID {
 	board := mustGetBoard(bid)
 	thread := mustGetThread(board, threadID)
 
+	assertThreadVisible(thread)
+
 	// TODO: Assert thread is not locked
 	// TODO: Assert that caller is a board member (when board type is invite only)
-
 	var reply *Post
 	if replyID == threadID {
 		// When the parent reply is the thread just add reply to thread
 		reply = thread.AddReply(caller, body)
 	} else {
 		// Try to get parent reply and add a new child reply
 		post := mustGetReply(thread, replyID)
+		assertThreadVisible(thread)
```

Comment:
```suggestion
		assertThreadVisible(post)
```

It might be good, a little bit redundant though, to define a `assertReplyVisible()` assertion because otherwise the assertion text would mention thread instead of reply, because replies could be flagged individually.